Utils/Dataflow/skel/stage.py

```diff
@@ -0,0 +1,66 @@
+#!/bin/env python
+"""
+DKB Dataflow stage XXX (StageName).
+
+Stage short description
+
+Authors:
+  Author Name (author@cern.ch)
+"""
```

Comment:
unknown author? ))

Reply:
Yeah, I didn\`t want my name to appear in all the lated scripts as the one and only author\`s name. What about yours? ;-)

Reply:
But it's your script ) so, I think there is no problem.